packages/astro/src/components/HeaderNav.astro

```diff
@@ -24,13 +28,22 @@ const currentPage = fullPath.slice(fullPath.lastIndexOf("/") + 1);
 		</li>
 	</ul>
 	<ul>
-		{
-			routes.map(([page, label]) => (
-				<li>
-					<a href={page} aria-current={currentPage === page ? "page" : false}>{label}</a>
-				</li>
-			))
-		}
+		{routes.map(route => (
+            <li>
+                {route.pages ? (
+                    <div class="dropdown">
+                        <a href={route.page} aria-current={currentPage === route.page ? "page" : false}>{route.label}</a>
+                        <div class="dropdown-content">
+                            {route.pages.map(page => (
+                                <a href={page.page} aria-current={currentPage === page.page ? "page" : false}>{page.label}</a>
+                            ))}
+                        </div>
+                    </div>
+                ):(
+                    <a href={route.page} aria-current={currentPage === route.page ? "page" : false}>{route.label}</a>
+                )}
```

Comment:
Rather than repeating `<a href={route.page} ...`, I wonder if that could always be shown, and then do something like `{route.pages?.length > 0 && <div class="dropdown-content"> ...` to optionally include the dropdown.  The `.dropdown` styles would probably need to be added to the `<li>` or `<a>` in that case.

Reply:
@fwextensions Yes we can certainly change that code. I like the more compact version. I have pushed an update to this. I have wrapped the dropdown class around a conditional however since all list items would not contain a dropdown